lib/cloudregister/registerutils.py

```diff
@@ -420,6 +420,7 @@ def find_repos(contains_name):
 # ----------------------------------------------------------------------------
 def get_activations():
     """Get the activated products from the update server"""
+    requests.packages.urllib3.disable_warnings()
```

Comment:
unnecessary line ?
we disable the warning for requests on [SMT](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/lib/cloudregister/smt.py#L56) already